ros_buildfarm/sourcedeb_job.py

```diff
@@ -116,20 +116,27 @@ def build_sourcedeb(sources_dir, os_name=None, os_code_name=None):
 
     def older_gbp():
         yield ('debian', 'jessie')
-        for distro_name in ['saucy', 'trusty', 'utopic', 'vivid',
+        for distro_name in ['saucy', 'utopic', 'vivid',
```

Comment:
fixed, i forgot to save the change locally. It's not explicitly relevant, since it would be excluded anyway.